src/foam/dao/history/HistoryDAO.js

```diff
@@ -43,12 +43,15 @@ foam.CLASS({
       name: 'javaExtras',
       buildJavaClass: function(cls) {
         cls.extras.push(
-          ` 
+          `
+            private static final PropertyPredicate propertyPredicate_ = new StoragePropertyPredicate();
```

Comment:
By convention, static variables should be capitalized if final, or end with two underscores __, if updatable or threadlocal. So these two should be capital/constantized and formatter_ should be formatter__.

See: https://github.com/foam-framework/foam2/blob/8ef0e293ad790ee1c73ef35f658f4f0e1ad52203/src/foam/util/StringUtil.java#L9

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/4385/commits/a90d41f3afb9235337b2cbd99f3008009e34a303.